lib/fedora_migrate/datastream_verification.rb

```diff
@@ -0,0 +1,43 @@
+module FedoraMigrate::DatastreamVerification
+  
+  attr_accessor :datastream
+
+  def valid? datastream=nil
+    @datastream = datastream || source
+    return true if no_checksum?
```

Comment:
If Fedora3 is configured to not do a checksum on the datastream, it's "none". This is to allow for other's who might have configured their repo differently, and I think thumbnails aren't checksummed.